src/resources/views/crud/form_content.blade.php

```diff
@@ -48,6 +48,58 @@ function initializeFieldsWithJavascript(container) {
       });
     }
 
+    /**
+     * Auto-discover first focusable input
+     * @param {jQuery} form
+     * @return {jQuery}
+     */
+    function getFirstFocusableField(form) {
+        return form.find('input, select, textarea, button')
+            .not('.close')
+            .not('[readonly]')
+            .filter(':visible:first');
+    }
+
+    /**
+     *
+     * @param {jQuery} firstField
+     */
+    function triggerFocusOnFirstInputField(firstField) {
+        if (firstField.hasClass('select2_field')) {
```

Comment:
I noticed while testing this that `select` and `fetch` relationship fields don't have the `select2_field` class. This is a custom backpack class that may be or not be in select2 fields.
I think a better solution here is checking for `select2-hidden-accessible` class, since that one guarantees us that the element is a select2.

Reply:
`select2_from_array` also have a different class. What they have in common is `select2-hidden-accessible` 👍